### PR TITLE
Non normative start

### LINKED ISSUE #2366: Poor structure of chapter "Packages"
The organization of the chapter _Packages_ is very poor, with pretty much all content hiding under the section _Motivation and Usage of Packages_.

This should be changed by extracting the _Motivation_ part to the chapter introduction (directly under \chapter), and breaking apart the rest of _Motivation and Usage of Packages_ to a sequence of sections.

### PR DESCRIPTION
**Pull request by HansOlsson**
Improve the structuring of chapters by stating that the initial text is non-normative; and should also restructure package-chapter.
Draft since there are some tricky remaining chapters.

Closes #2366

### CODE REVIEW

**Comment by HansOlsson** [comment]
Note on a missing commit: I believe that the classes-chapter should work as is. The introductory parts are as far as I understand duplicated.

File: chapters/operatorsandexpressions.tex [hunk 1]
```diff
@@ -86,6 +86,7 @@ \section{Operator Precedence and Associativity}\label{operator-precedence-and-as
 conditional & \lstinline!if $\mathit{expr}$ then $\mathit{expr}$ else $\mathit{expr}$! & \lstinline!if b then 3 else x!\\ \hline
 named argument & \lstinline!$\mathit{ident}$ = $\mathit{expr}$! & \lstinline!x = 2.26!\\ \hline
 \end{tabular}
+\label{tab:operator-precedence}
 \end{table}
 
 The conditional operator may also include elseif-clauses. Equality \lstinline!=! and
```
Comment (henrikt-ma) [inline chapters/operatorsandexpressions.tex:92]: This comment belongs to the current file, not this particular line.

There is still a `nonnormative` in the start of chapter 3:
> The abbreviated predefined type information below is given as background information for the rest of the presentation.

**Comment by henrikt-ma** [comment]
Shouldn't every chapter have a non-normative start?  Chapters missing one:
- _Equations_
- _Modelica Concrete Syntax_

**Review by henrikt-ma — APPROVED** [review]
This looks like a good start.  Approving, even though a few more things could be done, see comments.

**Comment by HansOlsson** [comment]
> Shouldn't every chapter have a non-normative start?

Ideally, yes. But it takes a bit of time to write a good introductions so I thought that could be done later.

File: chapters/operatorsandexpressions.tex (hunk 1 above)
Comment (HansOlsson) [reply chapters/operatorsandexpressions.tex:92]: Good point, I tried to figure out what it meant.
It refers to some "predefined type information below", and I cannot figure out what that refers to.
I thus decided to just remove it. It was introduced in Modelica 3.0.

Comment (henrikt-ma) [reply chapters/operatorsandexpressions.tex:88]: Fine.  Resolving.

**Pull request merged**